package.json

```diff
@@ -24,18 +24,15 @@
     "bucklescript"
   ],
   "scripts": {
-    "clean": "bsb -clean-world",
-    "build": "bsb -make-world",
-    "watch": "bsb -make-world -w",
+    "clean": "rescript clean",
+    "build": "rescript",
+    "watch": "rescript build -w",
     "test": "yarn build && yarn jest __tests__/*_test.bs.js",
     "atdgen": "cd __tests__ && esy atdgen -t test.atd && esy atdgen -bs test.atd"
   },
   "devDependencies": {
     "@glennsl/bs-jest": "^0.7.0",
-    "bs-platform": ">= 4.0.0"
-  },
-  "peerDependencies": {
-    "bs-platform": ">= 4.0.0"
+    "rescript": "^9.1.4"
```

Comment:
Also, I am not sure how this would affect projects that used Melange instead. 😕

Reply:
it makes it _better_, because there are no peer dependencies. Both ReScript and Melange users linking to this project have been dealing with a peer dependency warning for a while.

Reply:
> Both ReScript and Melange users linking to this project have been dealing with a peer dependency warning for a while.

I think this warning problem is more related to this library not following the major version releases of bs-platform / rescript.

re: Melange, I was thinking the scenario where someone working on a Melange project would bring this library using esy, then because rescript repo used to contain some `esy.json` files, it would trigger whole build of ReScript.

But this is a weird scenario. I think it's reasonable to expect that anyone using a dep like `bs-atdgen-codec-runtime` will use yarn or npm for JS deps, not esy. Also, I just realized there are no more esy.json files in rescript repo.